pkg/openSearch/esextension/scriptextension.go

```diff
@@ -22,6 +37,22 @@ func (a *ScriptedSumAggregation) Map() map[string]interface{} {
 	}
 }
 
+// ScriptedSumAgg is a function that creates a new instance of ScriptedSumAggregation.
+// It takes the name and script as parameters and returns a pointer to the ScriptedSumAggregation struct.
+// Example usage:
+//
+//	  a := ScriptedSumAgg("testName", "testScript")
+//	  if a.Name() != "testName" {
```

Comment:
this example looks really weird to me :-D
I would either remove it, or just take ```a := ScriptedSumAgg("testName", "testScript")``` and then write down the JSON output, removal preferred :-)